src/Components/App/App.js

```diff
@@ -6,28 +6,50 @@ import Settings from '../Settings/Settings'
 import About from '../About/About'
 import Stats from '../Stats/Stats'
 import PageNotFound from '../PageNotFound/PageNotFound'
-import './App.scss'
 import { enableNotifications } from '../../helpers/notificationHelpers'
-import { ToastContainer } from 'react-toastify';
+import { ToastContainer } from 'react-toastify'
+import './App.scss'
+
 
 function App() {
   enableNotifications()
+
+  const toggleTimerView = (hide) => {
+    const homeContainer = document.getElementById('HomeContainer')
```

Comment:
I'm actually surprised this works, because research I've done has suggested that if you use getElementById in React, you'd get the error `TypeError: document.getElementById(...) is null` - I think maybe the more conventional way to reference a DOM node in React is by using a ref (there's a useRef hook too). If this works, though, I'm all for leaving it. Just an observation!

Reply:
Aha! I hath never seen that error, I decided to try this based on the that way index.js gets the 'root' element. I wonder is that error would show up if this was done in JSX?

Reply:
Also - would you mind giving an example of how to use a ref for this? I hath never tried that before and would be curious to see it.

Reply:
@Abdeboskey Ah you might be right that that error would show up if this was done in JSX - that makes sense. I haven't used refs myself either yet, but was just looking at a few resources I found - React docs are a good place to start for the useRef hook